.travis.yml

```diff
@@ -90,7 +91,9 @@ addons:
       - powershell
     packages:
       - ccache
+      - cpanminus
       - ninja
+      - perl
```

Comment:
osx job passed so it's safe to assume that the perl provider supports recent releases only for osx.

Reply:
It works at least all the way back to perl 5.14, but should work for perl 5.12 and 5.10 as well:

https://dev.azure.com/jacquesgermishuys/p5-Neovim-Ext/_build/results?buildId=183&view=results

Reply:
Can you investigate this in your Azure pipeline by creating a new job that doesn't use perlbrew?

Reply:
I've tested this locally. Seems like the required perl headers are missing by default on macOS. They would need to be installed to use the system perl.

https://stackoverflow.com/questions/52682304/fatal-error-extern-h-file-not-found-while-installing-perl-modules
https://developer.apple.com/documentation/xcode_release_notes/xcode_10_release_notes#3035624

IMHO I think using perl from homebrew would be the simplest.